### DISCUSSION TITLE
Alerts API showing stale data

### DISCUSSION THREAD

**Discussion by tmabell** [weather-gov/api#782] [Q&A]
Edit: issue resolved

Since yesterday when the Winter Storm Watch was upgraded to a Winter Storm Warning, the API has been producing stale data.  See the link to the API for St. Joseph County, IN.  The Watch was upgraded yesterday AM but still shows active, even after expiration.

https://api.weather.gov/alerts/active.atom?point=41.6479%2C-86.1293

**Comment by dwhitemv25** [comment]
17 hours later, it seems to have disappeared... the link returns no alerts now. Could have been a coding error somewhere that didn't upgrade all of the watches. 

Looking at just the `/alerts` endpoint for your point, here's the visible VTEC codes:

```
            "/O.CAN.KIWX.WS.W.0003.000000T0000Z-241203T1500Z/"
            "/O.CON.KIWX.WS.W.0003.000000T0000Z-241203T1500Z/"
            "/O.CON.KIWX.WS.W.0003.000000T0000Z-241203T1500Z/"
            "/O.CON.KIWX.WS.W.0003.241202T2000Z-241203T1500Z/"
            "/O.UPG.KIWX.WS.A.0003.241202T1500Z-241203T1500Z/"
            "/O.NEW.KIWX.WS.W.0003.241202T2000Z-241203T1500Z/"
            "/O.EXA.KIWX.WS.A.0003.241202T1500Z-241203T1500Z/"
```

Nothing there seems abnormal. I could try to resolve the expiredReferences, but it'll have to wait till tomorrow.

**Reply by tmabell** [reply]
We may never know because everything returned to normal when the event (003) was cancelled for the remainder of the CWA. That was about 11AM or so.

**Reply by dwhitemv25** [reply]
Sadly, it would have been helpful to have a snapshot of the data returned from the query when the issue was occurring.

**Reply by akrherz** [reply]
@dwhitemv25 FWIW, I looked into this yesterday and observed the stale data in the API and did not find any issues with the VTEC product lifecycle to resolve why the stale data was there...

**Reply by sullynole** [reply]
Did anyone catch the urn:oid that was seen on the active endpoint?  Curious if it was this:
https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0.81848c60ad518c9da17e4fbc4b85d3d3eb818fdc.001.1

The urgency on that one is "Past" which is different from the original watch alert which has replacedBy and replacedAt the bottom
https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0.da2899f262c29bf644d10c3facf9ce556ae3a185.001.1

**Comment by tmabell** [comment]
I wish I had saved the entire message returned by the API but I did not.  I do however, remember seeing "Past" in the text.  If it happens again I will archive everything.
